internal/proxy/handler.go

```
@@ -62,6 +63,18 @@ func NewHandler() (*Handler, error) {
 
 	twitchService := services.NewTwitchService(cfg.TwitchClientID, cfg.TwitchClientSecret, cfg.TwitchTimeout)
 	proxyRequests := NewProxyRequests(cfg, twitchService)
+
+	// read magic-link JSON blob from SSM at req time otherwise fallback to .env value
+	if cfg.MagicLinkSSMParam != "" {
+		store, storeErr := magiclink.NewStore(context.Background(), cfg.MagicLinkSSMParam)
+
+		if storeErr != nil {
+			log.Printf("magic-link SSM init failed, falling back to env var %w", storeErr)
```

golangci-lint (2.12.2)

[error] printf: log.Printf does not support error-wrapping directive %w

(govet)